src/View/AjaxView.php

```diff
@@ -101,7 +101,11 @@ public function render($view = null, $layout = null) {
 		if (isset($this->viewVars['_serialize'])) {
 			$response = $this->_serialize($response, $this->viewVars['_serialize']);
 		}
-		return json_encode($response);
+		$result = json_encode($response);
+		if (json_last_error() != JSON_ERROR_NONE) {
```

Comment:
That's better, even while it shouldn't make any difference while it compares int vs int.